pkg/cli/admin/upgrade/status/examples/4.15.0-ec2-b02-cos-not-annotated.detailed-output

```diff
@@ -13,18 +13,19 @@ ip-10-0-92-180.us-east-2.compute.internal   Outdated     Pending   4.15.0-ec.1
 
 = Worker Upgrade =
 
-= Worker Pool =
-Worker Pool:     worker
-Assessment:      Pending
-Completion:      0%
-Worker Status:   3 Total, 3 Available, 0 Progressing, 3 Outdated, 0 Draining, 0 Excluded, 0 Degraded
+Worker Pool   Assessment   Completion   Status
+worker        Pending      0%           3 Total, 3 Available, 0 Progressing, 3 Outdated, 0 Draining, 0 Excluded, 0 Degraded
+zbeast        Completed    NaN%         0 Total, 0 Available, 0 Progressing, 0 Outdated, 0 Draining, 0 Excluded, 0 Degraded
```

Comment:
Given we link to the OTA-1277 card can we handle the empty case nicer? Assessment `Completed` is fine I guess, but we could say `Scaled to zero`, `Downscaled` or `Empty` instead. Let's get rid of the `NaN` too. We could also try pruning the `Status` to just `0 Total` but maybe only for this case.